src/credentials/CHIPCertToX509.cpp

```diff
@@ -494,7 +496,7 @@ static CHIP_ERROR DecodeConvertExtendedKeyUsageExtension(TLVReader & reader, ASN
             certData.mKeyPurposeFlags.Set(static_cast<KeyPurposeFlags>(0x01 << (keyPurposeId - 1)));
         }
 
-        VerifyOrExit(nextRes == CHIP_END_OF_TLV, err = nextRes);
+        VerifyOrExit(err == CHIP_END_OF_TLV, /* returns err */);
```

Comment:
There is a comma too many here I think? It's confusing.

Reply:
I realized that `reader.VerifyEndOfContainer()` is the correct function to use in this case. Replaced all instances accordingly.